src/Events/Migration/v0.21/01-setup-table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS events.events (
+    sequenceno BIGSERIAL,
+    cloudevent JSONB NOT NULL,
+    registeredtime timestamptz default (now() at time zone 'utc'),
+    CONSTRAINT events_pkey PRIMARY KEY (sequenceno)
```

Comment:
Is this table now using the same sequence as the app events table?

Reply:
no, the contraint was renamed for events_app table